css/joomla.css

```diff
@@ -375,3 +375,13 @@ ul#civicrm-menu li#crm-qsearch {
 .crm-container .disabled {
   font-weight: normal;
 }
+
+#crm-container .crm-container .form-layout td.label {
```

Comment:
I think this should be on two lines to get both cases, like 

    #crm-container .form-layout td.label,
    .crm-container .form-layout td.label {